api/v1alpha1/nodeobservabilitymachineconfig_types.go

```diff
@@ -40,6 +40,9 @@ type NodeObservabilityMachineConfigStatus struct {
 
 	// UpdateStatus contains of the status of the MCP update
 	UpdateStatus ConfigUpdateStatus `json:"updateStatus,omitempty"`
+
+	// Check if the MachineConfig is ready
+	Ready bool `json:"ready,omitempty"`
```

Comment:
I didnt see where this flag would be updated by a controller.
Could it be that this flag is the one we're supposed to look at instead of calling `r.MCOReconciler.CheckMCPUpdateStatus` on nodeobservabilityrun controller?

Reply:
Yes should be removed as it is not used. I just forgot to do it. well spotted